test/wdio/global-script/global-script.test.tsx

```diff
@@ -15,4 +17,29 @@ describe('global script', () => {
     const renderedDelay = parseInt(text.slice('I am rendered after '.length));
     expect(renderedDelay).toBeGreaterThanOrEqual(1000);
   });
+
+  it('logs error when component with invalid runtime is loaded', async () => {
+    /**
+     * Fetching logs like this only works in Chromium. Once WebdriverIO v9 is released there
+     * will be easier primitives to fetch logs in other browsers as well.
+     */
```

Comment:
we might want to create a JIRA ticket for upgrading to WebdriverIO v9 and then leave a TODO here to remove the `browser.isChromium` check